rb-files/decorator.rb

```diff
@@ -0,0 +1,23 @@
+require './nameable'
```

Comment:
- [ ] For ruby best practices and to fix the below error I am getting when trying to test your app, kindly make sure you update the way you import your files. As all of them are in the same directory you should use `require_relative` instead of `require` when you import files. `require` is used to import files of different directory. Make sure you update every where you use `require` to `require_relative` in order to fix the below error. **Make sure to apply this approach in all your files in the same directory**
```ruby
# For example

require_relative './nameable'

 class Person < Nameable
  def initialize(age, name = 'unknown', parent_permission: true)
    ...
  end
```
![Screenshot from 2023-04-11 06-52-12](https://user-images.githubusercontent.com/83042742/231139055-19228c75-9bc1-4210-9afc-58b132177649.png)